tools/i18n/makepot.php

```diff
@@ -9,11 +9,11 @@
 
 /**
  * Class to create POT files for
- *  - WordPress 3.4+
- *  - WordPress plugins
- *  - WordPress themes
+ *  - ClassicPress 3.4+
```

Comment:
ClassicPress 3.4+ doesn't exist yet, should probably remain WordPress?

Reply:
Let's leave all of this alone for now, and resolve separately.  See #45.